backend/test/modules/player-profile/core/application/SendEmailAfterProfileWasCreated.spec.ts

```diff
@@ -1,16 +1,17 @@
 import { PlayerProfileWasCreated } from '../../../../../src/modules/player-profiles/core/domain/event/PlayerProfileWasCreated';
-import { CommandPublisher } from '../../../../../src/shared/core/application/command/CommandBus';
-import { SendEmailAfterPlayerProfileWasCreatedEventHandler } from '../../../../../src/modules/player-profiles/core/application/event/SendEmailAfterPlayerProfileWasCreatedEventHandler';
+import { CommandBus } from '../../../../../src/shared/core/application/command/CommandBus';
 import { SendEmail } from '../../../../../src/modules/email-sending/core/application/command/SendEmail';
-import { testModuleCore, TestModuleCore } from '../../../../test-support/shared/core/TestModuleCore';
-import { InMemoryPlayerProfileRepository } from '../../../../../src/modules/player-profiles/infrastructure/repository/inmemory/InMemoryPlayerProfileRepository';
-import { PlayerProfilesModuleCore } from '../../../../../src/modules/player-profiles/core/PlayerProfilesModuleCore';
+import { testPlayerProfileModule } from './TestPlayerProfileModule';
 
 describe('Send an email to player', () => {
   it('When player profile is created, then execute command for email sending', async () => {
     //Given
     const currentTime = new Date();
-    const playerProfileModuleCore = testPlayerProfileModule_2(currentTime);
+    const commandBus: CommandBus = {
+      registerHandler: jest.fn(),
+      execute: jest.fn(),
+    };
+    const playerProfileModuleCore = testPlayerProfileModule(currentTime, commandBus);
```

Comment:
Ach, to tak!

Reply:
no tak to bym nie wpadł, żeby dodać parametr do testowej metody module core :-P